build.gradle.kts

```diff
@@ -21,7 +24,9 @@ infra {
 }
 
 allprojects {
+    logger.info("Using Kotlin $kotlin_version for project $this")
```

Comment:
please pass the compiler flag -version to log it

Reply:
Done